README.md

```diff
@@ -3,25 +3,23 @@
 ## Learning Goals
 
 - Describe the functionality of arguments
-- Define methods that accept arguments
+- Define a method that accepts arguments
 - Recognize the difference between arguments and parameters
 
 ## Introduction
 
-In the real world, many interactions are customized or specific to our
-day-to-day experiences. For example, if you go to a coffee shop, they may take
-down your name to call up your order when it's ready. If visit a doctor's
-office, they may call your name when it's your turn to be seen. Variables that
-we experience in real life often need to be translated into to programs.
+In the real world, many interactions have a "template" behavior that is applied
+to every interaction, but they change only slightly depending on a tiny factor.
 
-In the example of the doctor's office or the coffee shop they have a "template"
-behavior that they apply to everyone, but that they change only slightly
-depending on the particular customer or patient. Methods can also be written to
-have a "template" of shared activity but which can be changed just a little bit
-depending on some particular data. The particular bits that make the change are
-called "arguments."
+For example, if you go to a coffee shop, they may take down **your** name to call up
```

Comment:
let's remove the modal "may." We're speaking to a contrived reality. It reads easier w/o it.